rpcserver.go

```diff
@@ -379,29 +379,57 @@ func (r *rpcServer) MintAsset(ctx context.Context,
 				update.Error)
 		}
 
+		batches, err := r.cfg.AssetMinter.ListBatches(update.BatchKey)
```

Comment:
We can get this via the `update` channel above instead. That way, we don't need to scan through the results once we allow more than one pending batch.